src/Check/StreamWrapperExists.php

```diff
@@ -25,29 +21,15 @@
  */
 class StreamWrapperExists extends AbstractCheck implements CheckInterface
 {
-    /** @var array|Traversable */
+    /** @var array<string> */
     protected $wrappers;
 
     /**
-     * @param  string|array|Traversable $wrappers Stream wrapper name or an array of names
+     * @param  string|array<string> $wrappers Stream wrapper name or an array of names
      * @throws InvalidArgumentException
      */
-    public function __construct($wrappers)
+    public function __construct(string|array $wrappers)
```

Comment:
Any reason for removing Traversable?

Reply:
`if (count($missingWrappers) === 1) {` used below